### PR TITLE
eservice-enclave-info script

### PR DESCRIPTION
**Pull request by bvavala**
Signed-off-by: Bruno Vavala <bruno.vavala@intel.com>

### CODE REVIEW

File: eservice/pdo/eservice/pdo_enclave.py [hunk 1]
```diff
@@ -192,6 +193,21 @@ def get_enclave_basename():
...
+
+def get_enclave_epid_group():
+    global _epid_group
+    _epid_group = _pdo.get_epid_group()
+    return _epid_group
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+
+def dump_enclave_ias_settings():
```
Comment (EugeneYYY) [inline eservice/pdo/eservice/pdo_enclave.py]: New function get_enclave_epid_group() doesn't seem to be used. Is it an omission or it is intended for the future use?

Comment (EugeneYYY) [reply eservice/pdo/eservice/pdo_enclave.py]: My apologies - wrong copy and paste. I meant dump_enclave_is_settings()

Comment (bvavala) [reply eservice/pdo/eservice/pdo_enclave.py]: It is used in the eservice-enclave-info script.

File: eservice/pdo/eservice/scripts/EServiceEnclaveInfoCLI.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    # enclave configuration is in the 'EnclaveConfig' table
+    try :
+        logger.debug('initialize the enclave')
+        pdo_enclave_helper.initialize_enclave(config.get('EnclaveModule'))
+        
+        logger.info("MRENCLAVE: " + pdo_enclave.get_enclave_measurement())
+        logger.info("BASENAME: " + pdo_enclave.get_enclave_basename())
+        logger.info("EPID Group: " + pdo_enclave.get_enclave_epid_group())
+        pdo_enclave.dump_enclave_ias_settings()
+    except Error as e :
```
Comment (cmickeyb) [inline eservice/pdo/eservice/scripts/EServiceEnclaveInfoCLI.py]: Why use the string append for logger messages? We have printf-style formatting for the messages that is used consistently through the code.

File: eservice/pdo/eservice/scripts/EServiceEnclaveInfoCLI.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    parser.add_argument('--config', help='configuration file', nargs = '+')
+    parser.add_argument('--config-dir', help='configuration file', nargs = '+')
+
+    parser.add_argument('--identity', help='Identity to use for the process', required = True, type = str)
+
+    parser.add_argument('--logfile', help='Name of the log file, __screen__ for standard output', type=str)
+    parser.add_argument('--loglevel', help='Logging level', type=str)
+
+    options = parser.parse_args()
+
```
Comment (cmickeyb) [inline eservice/pdo/eservice/scripts/EServiceEnclaveInfoCLI.py]: Presumably there is sufficient support for parameters to run the script before the configuration is complete?

File: eservice/pdo/eservice/utility/ias_client.py [hunk 4]
```diff
@@ -31,28 +31,35 @@ class IasClient(object):
...
         self._timeout=300
 
+    def print_ias_settings(self):
+        logger.info("IAS settings:")
+        logger.info("Proxy: " + self._proxies["https"])
+        logger.info("SPID: " + self._spid)
+        logger.info("URL: " + self._ias_url)
+        logger.info("SpidCert: " + self._cert)
+
     def get_signature_revocation_lists(self,
```
Comment (cmickeyb) [inline eservice/pdo/eservice/utility/ias_client.py]: please don't use log messages for output that needs to be processed. if you are expecting a script to process the output then just print the output.

**Comment by byron-marohn** [comment]
Rolled into #66

**Pull request closed without merging**